Apps/W1/Shopify/app/src/Base/Tables/ShpfyLanguage.Table.al

```diff
@@ -0,0 +1,72 @@
+namespace Microsoft.Integration.Shopify;
+
+using System.Globalization;
+
+table 30154 "Shpfy Language"
+{
+    Caption = 'Shopify Language';
+    DataClassification = CustomerContent;
+
+    fields
+    {
+        field(1; "Shop Code"; Code[20])
+        {
+            Caption = 'Shop Code';
+            DataClassification = CustomerContent;
+            Editable = false;
+            TableRelation = "Shpfy Shop";
+        }
+
+        field(2; Locale; text[2])
```

Comment:
I don't know how it's in the cloud, but Text fields in primary key are evaluated as Code in primary keys depending on the SQL setting.
Is it important that the values are a Text and not a Code?

There has been a thread on this topic on yammer some months ago, but the search isn't working today and I cannot find this thread right now.

Reply:
Huh, I did not know about that behavior...
In this case, I need the locale to be in lowercase, as it's case-sensitive in Shopify, and it rejects a translation update otherwise. 
Technically, I can work around it, and transform it to lowercase before I send the update, but on a local Sandbox container, the Text key was maintained as text.

Reply:
Here some links on this topic:
https://github.com/StefanMaron/BusinessCentral.LinterCop/discussions/459
https://www.yammer.com/dynamicsnavdev/threads/2583811689037824